### PR TITLE
LGA-2647 : ✏️ Update the Online Safety guideline

### PR DESCRIPTION
**Pull request by sarahyjja**
## What does this pull request do?

- Update the content of the Online Safety guideline in the footer.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/online-safety.html [hunk 1]
```diff
@@ -3,43 +3,61 @@
...
 
-  <h2 class="govuk-heading-l">{{ _('Browsing in private') }}</h2>
+  <h2 class="govuk-heading-l">{{ _('How the ‘Exit this page’ button works on this website') }}</h2>
   <p class="govuk-body">
-    {{ _('You can also look at a website without information like cookies being stored by using the private browsing setting
-    in your browser. Go to the menu, click on ‘File’ and choose:') }}
+    {{ _('There is an ‘Exit this page’ button on some pages. If you click this button, all the information you’ve entered will be deleted. This website will close.')}}
   </p>
+  <p class="govuk-body">
+    {{ _('You’ll automatically go to the Google search homepage:') }} <a class="govuk-link" href="https://www.bbc.co.uk/weather" rel="external">{{ _('https://www.bbc.co.uk/weather') }}</a>
```
Comment (psweeting1) [inline cla_public/templates/online-safety.html:42]: You're still saying they're going to Google search homepage, when it's the BBC Weather.

`You'll automatically go to the BBC Weather homepage:`

Comment (sarahyjja) [reply cla_public/templates/online-safety.html:42]: Thanks Pat, missed that one! :)

**Review by psweeting1 — APPROVED** [review]
LGTM. 

Double check with Rez, but you can't merge this until the exit page button work is also ready to deploy. Don't want to merge this is in early when there isn't an exit page button in the actual website.

**Review by TawneeOwl — APPROVED** [review]
LGTM 🚀

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**